app/services/payment_requests/payments/adyen_service.rb

```diff
@@ -51,6 +52,7 @@ def update_payment_status(provider_payment_id:, status:, metadata: {})
 
         update_payable_payment_status(payment_status: payable_payment_status)
         update_invoices_payment_status(payment_status: payable_payment_status)
+        update_invoices_paid_amount_cents(payment_status: payable_payment_status)
```

Comment:
Out of the scope of this PR, but seeing this code repeated 5 times clearly shows that we should refactor these services to factorize the common logic in a root service and extract the logic specific to the payment provider in a sub services (in a similar way of what was done for payment creation)